sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -181,3 +183,25 @@ case class DescribeCommand(
       child.output.map(field => Row(field.name, field.dataType.toString, null))
   }
 }
+
+/**
+ * :: DeveloperApi ::
+ */
+@DeveloperApi
+case class DDLDescribeCommand(
+    relation: BaseRelation,
+    override val output: Seq[Attribute],
+    isExtended: Boolean) extends RunnableCommand {
+
+  override def run(sqlContext: SQLContext) = {
+    val rows = new ArrayBuffer[Row]()
+    rows += Row("# col_name", "data_type", "comment")
```

Comment:
@chenghao-intel 
ok, I will remove it.
If future support `partition table` here, I will add it like hive does:
```
# Partition Information          
# col_name              data_type               comment                              
patition_col_name               string                  None  
```